addons/mail/models/mail_thread.py

```diff
@@ -4478,6 +4479,8 @@ def _get_mail_thread_data(self, request_list):
             res['activities'] = self.with_context(active_test=True).activity_ids.activity_format()
         if 'attachments' in request_list:
             res['attachments'] = self._get_mail_thread_data_attachments()._attachment_format()
+            res['areAttachmentsLoaded'] = True
+            res['isLoadingAttachments'] = False
```

Comment:
These are pure JS concerns, I don't like to have them in python.
For these specifically it seems fine to have extra processing around the RPC.